test/archethic/reward/scheduler_test.exs

```diff
@@ -156,6 +156,29 @@ defmodule Archethic.Reward.SchedulerTest do
               }} = :sys.get_state(pid)
     end
 
+    test "should wait for :node_down message to stop the scheduler" do
```

Comment:
I don't understand this test, you may need to send a node_up and then ensure it stops when sending a node_down